subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -1449,6 +1467,7 @@ static void scene_recall(struct bt_mesh_model *model, const uint8_t data[],
 		};
 
 		ctrl_disable(srv);
+		schedule_resume_timer(srv);
```

Comment:
Is there an idea to move this function call into ctrl_disable?

Reply:
We cannot, as that would start the timer on the model reset callback :slightly_frowning_face: